bin/npg_manifest4study

```diff
@@ -118,16 +135,15 @@ sub main {
     };
   }
 
-  if (! keys %{$files} ) {
-    $logger->warn('Due to previous errors, nothing to report, exiting');
+  if (not keys %{$files} ) {
+    $logger->warn('Due to previous errors, nothing to report');
+    $send_empty_manifest->();
```

Comment:
Is this the case where our internal errors mean no manifest is generated? *If so* then I think we should not send the manifest here. To give missing context to the request for this "empty manifest" functionality, the empty manifest is meant to indicate that all is well our side even though no files have been uploaded to report in a manifest.

Reply:
This is fixed in commit 3c060b9ef4cc024ca0a755d3afa0ef81c034059c